### PR TITLE
Add tty working dir and hold

### PR DESCRIPTION
**Pull request by leb-kuchen**
See #179 for more reasons why these options are useful.
For the working directory, if the shell does not provide to execute a command at startup, you will need to something like `bash -c "cd somewhere; exex bash"`.
For hold, I think true should be the default , since like in the issue above, the terminal will exit if it also a  default, which means you will have to edit or remove the config file manually and have no GUI. 
Hold probably should be a drop down like in gnome terminal, for now just a showcase.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -899,11 +911,29 @@ impl App {
...
                                 ),
                             ),
+                        )
+                        .add(
+                            widget::row::with_children(vec![
+                                widget::column::with_children(vec![
+                                    widget::text(fl!("hold")).into(),
+                                    widget::text::caption(fl!("remain-open")).into(),
+                                ])
+                                .spacing(2)
```
Comment (jackpot51) [inline src/main.rs:926]: Is there a reason why the spacing is 2 here and space_xxxs is commented out?

**Comment by jackpot51** [comment]
This looks good if that manual spacing item can be resolved.

**Comment by leb-kuchen** [comment]
I did some experimenting and thought the  lowest spacing of 4 is too much.  Anyway does your UX team prefer a dropdown over a toggler, cause I think these options can be explained more cleanly?

**Comment by jackpot51** [comment]
I think either a toggler or a dropdown is fine. The lowest spacing of 4 was chosen to ensure that fractional scaling in multiples of 25% always ends up with an integer.

**Review by jackpot51 — APPROVED** [review]

**Comment by jackpot51** [comment]
This looks good enough to me now. We may adjust later based on UX feedback.

**Pull request merged**